thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-details/subscription-group-details.component.tsx

```diff
@@ -23,6 +23,8 @@ export const SubscriptionGroupDetails: FunctionComponent<SubscriptionGroupDetail
         const propertiesFormValues = {
             cron: subscriptionGroup.cron,
             name: subscriptionGroup.name,
+            notifyHistoricalAnomalies:
+                !!subscriptionGroup.notifyHistoricalAnomalies,
```

Comment:
nit: for better readability 
```suggestion
            notifyHistoricalAnomalies:
                Boolean(subscriptionGroup.notifyHistoricalAnomalies),
```

Reply:
Done